back/src/routes/book-router.ts

```diff
@@ -0,0 +1,151 @@
+import { Router } from "express";
+import { ClubModel } from "../models/club";
+import { BookModel, BookInterface } from "../models/book";
+
+const bookRouter = Router();
+
+// get all
+bookRouter.get("/", async (req, res) => {
+  const books = await BookModel.find();
```

Comment:
do we want to filter by the user's books?

Reply:
i dont think i save which books belong to each user
and if we'll do that, maybe it should be in the user's route?
should i also do get all books by club id?